### PR TITLE
feat: removed HTTP communication

### PR DESCRIPTION
**Pull request by Kapim**
- SocketIO's websockets are now the only way to transfer data to network applications
- arguments are now passed to the network application using the /command namespace (see register method)

### CODE REVIEW

File: era_5g_client/client_base.py [hunk 1]
```diff
@@ -230,43 +138,23 @@ def send_image_ws(self, frame: np.ndarray, timestamp: Optional[str] = None):
...
-            json=json,
-            headers={"Content-Type": "application/json"},
-        )
-
-        if not response.ok and self._json_error_event:
-            self._json_error_event(f"{response.reason} - {response.text}")
+        f = base64.b64encode(img_encoded)
+        self._sio.emit("image", {"timestamp": timestamp, "frame": f}, "/data")
 
     def send_json_ws(self, json: dict) -> None:
```
Comment (ZdenekM) [inline era_5g_client/client_base.py:145]: The annotation should be `Dict` for Python 3.8.

File: era_5g_client/client_base.py [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
 import numpy as np
-import requests
 import socketio
-from requests import Response
+from socketio.exceptions import ConnectionError  # type: ignore
 
 from era_5g_client.dataclasses import NetAppLocation
-from era_5g_client.exceptions import FailedToConnect, FailedToSendData, NetAppNotReady
+from era_5g_client.exceptions import FailedToConnect
+from era_5g_interface.dataclasses.control_command import ControlCmdType, ControlCommand  # type: ignore
```
[the remark this answers is not in the document]
Comment (Kapim) [reply era_5g_client/client_base.py:15]: tbh, I don't know :-) but without it, the test did not pass

File: era_5g_client/client_base.py [hunk 3]
```diff
@@ -7,12 +7,12 @@
 
 import cv2
 import numpy as np
-import requests
 import socketio
-from requests import Response
+from socketio.exceptions import ConnectionError  # type: ignore
```
[the remark this answers is not in the document]
Comment (Kapim) [reply era_5g_client/client_base.py:11]: done

File: era_5g_client/client_base.py (hunk 1 above)
Comment (Kapim) [reply era_5g_client/client_base.py:148]: fixed

File: mypy.ini [hunk 4]
```diff
@@ -36,4 +36,10 @@ ignore_missing_imports = True
...
 
 [mypy-socketio]
-ignore_missing_imports = True
\ No newline at end of file
+ignore_missing_imports = True
+
+[mypy-socketio.*]
+ignore_missing_imports = True
+
+[mypy-era_5g_interface.*]
```
Comment (ZdenekM) [inline mypy.ini:44]: @Kapim This should not be necessary for first-party code. I will take a look at it.

**Pull request merged**